### PR TITLE
Add opentx os x build vm and allow more cores to be used for fast mac…

### PR DESCRIPTION
**Pull request by schwabe**
…hines.

### CODE REVIEW

File: tools/build-companion.sh [hunk 1]
```diff
@@ -20,42 +20,45 @@ if [ "$3" == "" ]; then
    exit 1
 fi
 
+if [ "$JOBS" == "" ]; then
+    JOBS=2
+fi
```
Comment (projectkk2glider) [inline tools/build-companion.sh]: I would prefer if JOBS was an option to the script like in https://github.com/opentx/opentx/blob/next/tools/commit-tests.sh, but then you have to take care of the other two arguments also. Nothing critical if it stays the way it is here.

Comment (schwabe) [inline tools/build-companion.sh]: I can also remove that from this pull request for now until we have a better option

**Comment by projectkk2glider** [comment]
Man, you are to quick :stuck_out_tongue:

File: tools/build-companion.sh [hunk 2]
```diff
@@ -4,11 +4,28 @@
 set -e
 set -x
 
+JOBS=2
+
+while [ $# -gt 0 ]
+do
+  case "$1" in
+    --jobs=*)
+      CORES="${1#*=}";;
```
Comment (projectkk2glider) [inline tools/build-companion.sh]: My bad, change CORES to JOBS. Same problem two lines down.

Comment (schwabe) [inline tools/build-companion.sh]: I should have tested it

**Pull request merged**